### PR TITLE
Joe's diagnostic

### PR DESCRIPTION
**Pull request by joechristman**
New rhyme.txt file and answer in diagnostic.md

### CODE REVIEW

**Comment by joechristman** [comment]
none.

File: cli-diagnostic/rhyme.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+The rain in Spain falls mainly in the plain.
```
Comment (berziiii) [inline cli-diagnostic/rhyme.txt:1]: Correct.

File: diagnostic.md [hunk 2]
```diff
@@ -34,21 +34,15 @@ Open up this file in Atom, and write your answers below (where indicated).
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+Absolute path because it goes all the way to the root.
```
Comment (berziiii) [inline diagnostic.md:37]: Correct.

File: diagnostic.md [hunk 3]
```diff
@@ -34,21 +34,15 @@ Open up this file in Atom, and write your answers below (where indicated).
...
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+Absolute because it points to an external URL
```
Comment (berziiii) [inline diagnostic.md:41]: Correct.

File: diagnostic.md [hunk 4]
```diff
@@ -34,21 +34,15 @@ Open up this file in Atom, and write your answers below (where indicated).
...
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+relative because the file is local
```
Comment (berziiii) [inline diagnostic.md:45]: Correct.

**Comment by berziiii** [comment]
6 correct out of 6 possible. Great job!

**Pull request closed without merging**